docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -471,7 +466,47 @@ To encode a raw image:
    ```
 
 <a name="animation"></a>
-## Encoding an Animated GIF
+## Encoding an Animated GIF or WEBP
+
+To encode an animated GIF or WEBP image, follow these steps:
+
+1. Create an encoding handle using `image_util_anim_encode_create()`:
+
+   ```
+   image_util_anim_encode_h anim_encode_h = NULL;
+   ret = image_util_anim_encode_create(IMAGE_UTIL_ANIM_WEBP, &anim_encode_h);
+   ```
+
+2. Set the loop count, background color or lossless `image_util_anim_encode_set_loop_count()`, `image_util_anim_encode_set_background_color()` or `image_util_anim_encode_set_lossless()`:
```

Comment:
2. Set the loop count, background color or lossless `image_util_anim_encode_set_loop_count()`, `image_util_anim_encode_set_background_color()` or `image_util_anim_encode_set_lossless()`:

Can we say 
I might be wrong, but request you to check and modify if needed 
```suggestion
2. Set the loop count, the background color or the lossless compression properties with `image_util_anim_encode_set_loop_count()`, `image_util_anim_encode_set_background_color()` or `image_util_anim_encode_set_lossless()`: